test/Microsoft.ML.Tests/Transformers/ConvertTests.cs

```diff
@@ -78,8 +78,8 @@ public void TestConvertWorkout()
             var data = new[] { new TestClass() { A = 1, B = new int[2] { 1,4 } },
                                new TestClass() { A = 2, B = new int[2] { 3,4 } }};
             var dataView = ML.Data.ReadFromEnumerable(data);
-            var pipe = new TypeConvertingEstimator(Env, columns: new[] {new TypeConvertingTransformer.ColumnInfo("ConvA", DataKind.R4, "A"),
-                new TypeConvertingTransformer.ColumnInfo("ConvB", DataKind.R4, "B")});
+            var pipe = new TypeConvertingEstimator(Env, columns: new[] {new TypeConvertingEstimator.ColumnInfo("ConvA", DataKind.R4, "A"),
```

Comment:
>TypeConvertingEstimator [](start = 27, length = 23)

why not ml.Catalog.ConvertType?

Reply:
Sure! Let's do it :)

---
In reply to: [253204389](https://github.com/dotnet/machinelearning/pull/2365#discussion_r253204389) [](ancestors = 253204389)